utility/test/module_test.go

```diff
@@ -114,3 +114,10 @@ func createTestingGenesisAndConfigFiles(cfg modules.Config, genesisState modules
 		log.Fatal(err)
 	}
 }
+
+var actorTypes = []utilTypes.ActorType{
```

Comment:
1. Please move this to the top of the file. General best practice for all shared variables / constants.

2. In the actual sourcecode, you use `typesUtil.ActorType_value`. Why not either leverage the same pattern in tests or have this array be in the source code?